### ISSUE TITLE
Update the micro workflow

### ISSUE THREAD

**Issue by FayKanellidou** [Clinical-Genomics/cg_lims#511]
Lims Issue
---------------

For definitions of headings below see [Basic Concepts](https://atlas.scilifelab.se/infrastructure/lims/basic_concepts/).

**Work Flow:** Microbial WGS v7
**Protocol:** CG002 - Normalization of microbial samples, Pre Processing Microbial v1, Nextera Microbial WGS v4 
**Step:** CG002 - Normalization of microbial samples for sequencing, Reception Control, Normalization of microbial samples v2

**Description:** 
In connection with [this issue](https://github.com/Clinical-Genomics/audits/issues/128)

In CG002 - Normalization of microbial samples for sequencing please include a udf with the [Barcode document](https://atlas.scilifelab.se/production/lab/supporting_procedures/barcodes_at_cg/)

In the Reception Control please include a udf again with the Barcode document](https://atlas.scilifelab.se/production/lab/supporting_procedures/barcodes_at_cg/) and another one with the [microbial method document](https://atlas.scilifelab.se/production/lab/library_prep_methods/manual_preps/manual_nextera/)

In the Normalization of microbial samples v2 please add an extra option in the udf "Supporting Procedure Document" a manual option.

**Comment by idalindegaard** [comment]
I did the wished changes.

Note: We cannot add "Manual" as an option for the udf Supporting Procedure Document, since the Atlas method document script needs the title of an actual document that we have at CG. I made the udf "not required" so there is an option to fill it in.

**Extra:** Made a combo automation so that automations "Arnold Upload Sample", "Atlas Method Document" and "Check missing sample udfs and set Next Step Remove" can run at the same time when exiting Record Details in step Reception Control in Pre Processing Microbial v1. The new automation is called "Check missing sample udfs + set Next Step Remove + Arnold + Atlas Method Document micro"
